docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -304,6 +304,7 @@ The following command operators are supported:
     * **NR**: No :ref:`route configured <arch_overview_http_routing>` for a given request in addition to 404 response code, or no matching filter chain for a downstream connection.
     * **URX**: The request was rejected because the :ref:`upstream retry limit (HTTP) <envoy_v3_api_field_config.route.v3.RetryPolicy.num_retries>`  or :ref:`maximum connect attempts (TCP) <envoy_v3_api_field_extensions.filters.network.tcp_proxy.v3.TcpProxy.max_connect_attempts>` was reached.
     * **NC**: Upstream cluster not found.
+    * **DT**: Request or connection exceeded the downstream connection duration.
```

Comment:
Would you mind adding a link to the configuration option (for http connection manager and tcp proxy, as with URX). That way it's clear which configuration option this response flag is tied to.

Reply:
This flag only reproduced when the maximum duration exceeded for a request or connection. We can't configure it.

Reply:
And I think we should link to those configuration options from this description, similar to what was done for URX.

Reply:
I have added links.